local-dev.yml

```diff
@@ -86,7 +89,7 @@ services:
     volumes:
       - "postgis_data_vol:/var/lib/postgresql/data"
     ports:
-      - "54322:5432"
+      - "5432:5432"
     stdin_open: true
```

Comment:
the intent here is to avoid conflicts with other postgres processes, running on a local developer laptop. Please discuss the potential for keeping this on 54322;  imagine a developer, who is running postgres locally, via brew and always has 5432 or is working on another dev sample that requires 5432.

Reply:
Haha, thanks, I confused that when I made a mistake to change from 54322 to 5432 😂